README.md

```diff
@@ -110,7 +110,7 @@ python manage.py migrate wagtailcore 0058
 python manage.py migrate
 ```
 
-These commands revert the `wagtailcore 0059` migration back to `wagtailcore 0058` to prevent the table conflict from occurring. This step should hopefully be unnecessary after the bug is fixed.
+These commands revert the `wagtailcore` migrations back to `wagtailcore 0058` to prevent the table conflict from occurring. This step should hopefully be unnecessary after the bug is fixed.
```

Comment:
Technically we revert all migrations that are more recent than `0058`, not just 0059.